airgun/views/computeresource.py

```diff
@@ -6,8 +6,10 @@
     View,
     ConditionalSwitchableView,
 )
-from airgun.views.common import BaseLoggedInView, SearchableViewMixin
-from airgun.widgets import FilteredDropdown, ActionsDropdown, SatTable
+from airgun.views.common import BaseLoggedInView, \
+        SearchableViewMixin, SatTab, SatTable
```

Comment:
SatTable isn't a view, flake warning was actually right :) Pls revert this change and import it from `airgun.widgets` instead.
And btw try to avoid backslashes and use following format instead
```python
from airgun.views.common import (
    BaseLoggedInView,
    SearchableViewMixin,
    SatTab,
)
```
if it doesn't fit in 1 line